src/inspectorscripts.ts

```diff
@@ -119,6 +119,10 @@ def _jupyterlab_variableinspector_dict_list():
             return True
         if tf and isinstance(eval(v), tf.Variable):
             return True
+        if pd and pd is not None and (
+            isinstance(eval(v), pd.core.frame.DataFrame)
+            or isinstance(eval(v), pd.core.series.Series)):
+            return True
```

Comment:
I'm not sure of the purpose of this function, but if we're concerned with performance wouldn't caching the value `ev = eval(v)` and reusing `ev` be faster?